docs/faq.md

```diff
@@ -2,119 +2,160 @@
 
-There will also be a "Classic Text" block, which is virtually the same as the current editor, except in block-form. There’s also likely to be a very popular plugin in the repository to replace Gutenberg with the classic editor.
+We are looking at ways to make Gutenberg configurable for many use cases, including disabling different aspects (like blocks, panels, etc.).
+
+There is also be a "Classic Text" block, which is virtually the same as the current editor, except in block form. There’s also likely to be a very popular plugin in the repository to replace Gutenberg with the classic editor.
 
 ## How will custom TinyMCE buttons work in Gutenberg?
+
 Custom TinyMCE buttons will still work in the "Classic Text" block, which is a block version of the classic editor you know today.
 
 (Gutenberg comes with a new universal inserter tool, which gives you access to every block available, searchable, sorted by recency and categories. This inserter tool levels the playing field for every plugin that adds content to the editor, and provides a single interface to learn how to use.)
 
 ## How will shortcodes work in Gutenberg?
-Shortcodes will continue to work as they do now.
+`Shortcodes` will continue to work as they do now.
 
-However we see the block as an evolution of the shortcode. Instead of having to type out code, you can use the universal inserter tray to pick a block and get a richer interface for both configuring the block, and previewing it. We would recommend people eventually upgrade their shortcodes to be blocks.
+However we see the block as an evolution of the `shortcode`. Instead of having to type out code, you can use the universal inserter tray to pick a block and get a richer interface for both configuring the block and previewing it. We would recommend people eventually upgrade their `shortcodes` to be blocks.
 
 ## Should I move shortcodes to content blocks?
 We think so. Blocks are designed to be visually representative of the final look, and they will likely become the expected way in which users will discover and insert content in WordPress.
 
 ## Will Gutenberg be made properly accessible?
-Accessibility is not an afterthought. Although not every aspect of Gutenberg is accessible at the moment, we are tracking issues in https://github.com/WordPress/gutenberg/labels/Accessibility, and we are fixing them. We understand that WordPress is for everyone, and that accessibility is about inclusion. This is a key value for us.
 
-If you would like to contribute to the accessibility of Gutenberg, we could always use people to test, and contribute.
+Accessibility is not an afterthought. Not every aspect of Gutenberg is accessible at the moment, but we are tracking issues in https://github.com/WordPress/gutenberg/labels/Accessibility, and we are fixing them. We understand that WordPress is for everyone, and that accessibility is about inclusion. This is a key value for us.
```

Comment:
Wondering if we ought to be using more inline links `we are [tracking issues](https://...) and are`